src/code/bank0.asm

```diff
@@ -843,32 +843,35 @@ MarkTriggerAsResolved::
     pop  af                                       ; $0C7B: $F1
     ret                                           ; $0C7C: $C9
 
-ApplyMapFadeOutTransition::
+ApplyMapFadeOutTransitionWithNoise::
```

Comment:
Nitpick: as this function is only used for playing a stairs noise, it could be more appropriately named `ApplyMapFadeOutTransitionWithStairsNoise`

Reply:
Yes but after looking at all references it seems, that the StairsNoise is also use when it is not a transition with stairs.
Before that I had nearly your name choosen.